assets/sass/local.scss

```diff
@@ -247,3 +247,7 @@
     margin-bottom: govuk-spacing(3);
   }
 }
+
+.govuk-table__cell {
+ overflow-wrap: anywhere;
```

Comment:
Tried this out locally and it's causing words to break mid-way through where it shouldn't happen (see screenshot). Depends a lot on window size so you don't always see it. Might need to be more targeted to the specific columns/cells likely to contain this kind of data?


![Screenshot 2023-02-10 at 08 10 40](https://user-images.githubusercontent.com/1441286/218038755-9bd1bbf3-8c12-45a0-86f4-e8284d06e1dc.png)